packages/core/src/sharedWishlists/client/index.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Shared wishlists clients.
+ *
+ * @module sharedWishlist/client
+ * @category Shared Wishlist
+ * @subcategory Clients
+ */
+import { warnDeprecatedMethod } from '../../helpers';
+
+warnDeprecatedMethod(
+  '@farfetch/blackout-core',
+  '@farfetch/blackout-core/search/client',
```

Comment:
Is this correct?

Reply:
I have deleted this as it was unintended to be there!